merchants/verkkokauppa/helpers.py

```diff
@@ -39,9 +39,11 @@ def get_formatted_reservation_time(reservation: Reservation) -> str:
 
     preferred_language = reservation.reservee_language or "fi"
     weekday = localized_short_weekday(begin.weekday(), preferred_language)
+    date = "{d.day}.{d.month}.{d.year}".format(d=begin)
```

Comment:
🪄 `date = begin.strftime("%-d.%-m.%Y")` 🪄

Reply:
Initially I was going to use this solution but apparently this is [OS-specific solution](https://www.enthought.com/blog/no-zero-padding-with-strftime/). Looks like this does not work on Windows so that is why I chose to not use it. I'd prefer to use a solution that works on all OSs. What are your thoughts on this? ☺️

Reply:
It is good point indeed. However my two cents is that this app is supposed to be run in *nix based system and so our container images are linux images (distros varies).
I don't know how well this would run in windows but I guess there would be some oddities.

Reply:
This probably isn't issue in production since we know it is a *nix based system. I'm more worried about the future developer who uses Windows machine without Linux subsystem 😅  This is going to fail on their machine and same happens with unit tests. I read more about this and it looks like this is not only a Windows issue and can happen in *nix systems as well, depending on the setup/configuration. Very hazy thing overall 🤪 

So I think I'll keep the `.format` solution. It is a bit ugly but it is guaranteed to work on all machines and won't backfire in the future 😄